### PR TITLE
Add unit tests for auth feature

### PR DESCRIPTION
**Pull request by m-rudyk**
# Description

Refactor auth feature in order to cover with unit tests.
Move auth under entities folder;
Add unit tests to cover main functionality;

Fixes # ([245](https://github.com/Samsung/LPVS/issues/245), https://github.com/Samsung/LPVS/issues/242)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Code cleanup/refactoring
- [ ] Documentation update
- [ ] This change requires a documentation update
- [ ] CI system update
- [x] Test Coverage update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration. Below is just an example and you can delete it when you contribute.

**Test Configuration**:
* Java: v11
* LPVS Release: v1.x.x

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/test/java/com/lpvs/entity/auth/OAuthServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
+package com.lpvs.entity.auth;
```
Comment (o-kopysov) [inline src/test/java/com/lpvs/entity/auth/OAuthServiceTest.java:1]: Please add file header

File: src/main/java/com/lpvs/entity/auth/OAuthAttributes.java [hunk 2]
```diff
@@ -55,7 +56,8 @@ public enum OAuthAttributes {
         this.of = of;
     }
 
-    public static MemberProfile extract(String registrationId, Map<String, Object> attributes) {
+    public static MemberProfile 
+    extract(String registrationId, Map<String, Object> attributes) {
```
Comment (v-kerimov) [inline src/main/java/com/lpvs/entity/auth/OAuthAttributes.java:60]: Doesn't seem to be a substantial change.

**Review by v-kerimov — APPROVED** [review]
Auth part of the project's test coverage was 0% previously so this is a good addition.

File: src/main/java/com/lpvs/entity/auth/OAuthAttributes.java (hunk 2 above)
Comment (m-rudyk) [reply src/main/java/com/lpvs/entity/auth/OAuthAttributes.java]: thanks, good point.

**Review by o-kopysov — APPROVED** [review]
+1

**Pull request merged**